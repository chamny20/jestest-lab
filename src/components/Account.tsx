const Account = (props: { name: string; mbti: string }) => {
  return (
    <>
      <h2>{props.name}</h2>
      <p>{props.mbti}</p>
    </>
  );
};

export default Account;
