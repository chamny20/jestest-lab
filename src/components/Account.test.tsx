import Account from "./Account";
import { render } from "@testing-library/react";

test("matches snapshot", () => {
  const utils = render(<Account name="chaemin" mbti="estj" />);
  expect(utils.container).toMatchSnapshot();
});
