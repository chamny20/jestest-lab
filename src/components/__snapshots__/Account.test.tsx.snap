// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <h2>
    chaemin
  </h2>
  <p>
    estj
  </p>
</div>
`;
